gravityflow/gravityflowflowchart
# Fixed duplicate start step on Flowchart

## Description
[Issue# 55](https://github.com/gravityflow/backlog/issues/55)

## Testing instructions
1. Test a workflow with many different steps, ensure the flowchart doesn't display the box-shaped duplicate start step.
2. Test an older spec workflow with no start step listed, ensure the flowchart still displays steps correctly including the green pill-shaped start step.

## Automated Test Enhancements
N/A

## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
